test/renameKeyWith.js

```diff
@@ -0,0 +1,30 @@
+import { assert } from 'chai';
+import { concat } from 'ramda';
+
+import * as RA from '../src';
+
+describe('renameKeyWith', function () {
+  it('should rename object keys', function () {
+    assert.deepEqual(RA.renameKeyWith(concat('a'), 'A', { A: 1 }), {
+      aA: 1,
+    });
+  });
+
+  it('should not change other key-value pairs', function () {
+    assert.deepEqual(RA.renameKeyWith(concat('a'), 'A', { A: 1, B: 2 }), {
+      aA: 1,
+      B: 2,
+    });
+  });
+
+  it('should return an empty object when renames keys on null and undefined', function () {
+    assert.deepEqual(RA.renameKeyWith(concat('a'), 'A', null), {});
+    assert.deepEqual(RA.renameKeyWith(concat('a'), 'A', undefined), {});
+  });
+
+  it('should be curried', function () {
+    assert.deepEqual(RA.renameKeyWith(concat('a'))('A')({ A: 1 }), {
```

Comment:
Can we make this more explicit ?

Explicit different combination of calling signatures would be great. https://github.com/char0n/ramda-adjunct/blob/b4f2188b602fde9d0d9c221ce05280d46a4ac510/test/viewOr.js#L31

Reply:
Would it be fine to add all possible asserts to this test?

Reply:
Sure, check out the test for `viewOr` that I've attached. We want the same thing here.